assets/script.js

```diff
@@ -0,0 +1,435 @@
+// console.log("hello")
+
+// Targeting the banner box
+const bannerBox = document.getElementById("banner-box");
+
+// Targeting the start quiz button
+const startButton = document.getElementById("start-btn")
+
+// Targeting the main section
+const mainSection = document.getElementById("main")
+
+const resultDiv = document.querySelector("#result-div");
+const resultText = document.querySelector("#result-text");
+
+// const targeting the time
+const timerSpan = document.getElementById("time-span")
+
+// const formSection = document.getElementById("form-box");
+
+// question index
+let questionIndex = 0;
+let time = 60;
+let feedbackResults ={};
+let quizComplete = false;
+let correctAnswers = 0;
+let currentQuestion
+const scoresLSKey = "highScores";
+let gameOver = false;
+
+
+const handleTimerButton = () => {
+    console.log("start button clicked");
+    const updateTimerValue = () => {
+      // increase the  timer by 1
+      time -= 1;
+  
+      // set text to new timer figures
+      timerSpan.textContent = time;
+      if (quizComplete) {
+        clearInterval(timerId);
+        document.getElementById("timer").remove();
+      } else {
+  
+      // check if timer is 0
+      if (time <= 0) {
+        clearInterval(timerId);
+      }
+    }
+};
+  
+    // start the timer
+    const timerId = setInterval(updateTimerValue, 1000);
+    console.log(timerId);
+  };
+
+  startButton.addEventListener("click", handleTimerButton);
+
+  document.getElementById("time-span").addEventListener("click", () => {
+    time -= 1;
+  });
+
+
+
+// // an element for results ls
+// const highScore = {
+//     score: score,
+//   };
+
+// using local storage for the highscores
+const feedbackSoresLS = (key) => {
+    return JSON.parse(localStorage.getItem(key));
+  };
+
+
+const highscoresLS = () => {
+    return feedbackSoresLS(scoresLSKey);
+  };
+
+const highScoresInLS = () => {
+    localStorage.setItem(scoresLSKey, JSON.stringify([]));
+  };
+
+
+  const feedbackSoresToLS = (key, data) => {
+    localStorage.setItem(key, JSON.stringify(data));
+  }
+
+  const feedbackHighscoresLS = (data) => {
+
+    const highScores = highscoresLS();
+    // push into an array
+    highScores.push(data);
+    //write the updated array into local storage
+    writeToLS(scoresLSKey, highScores);
+  };
+
+  const HighscoresPage = () => {
+    //   from local storage
+    const feedbackHighscoresFromLS = highscoresLS ();
+
+    // check if it exist
+    if (!feedbackHighscoresFromLS) {
+        highScoresInLS();
+      }
+  };
+
```

Comment:
I think these are the high scores functions don't seem to be working. It might be worth looking at these to try and get them working as I was unable to add my high score and save it.